doc/_includes/tutorials/common/creating-a-project.md

```diff
@@ -18,14 +18,20 @@ If you choose to set a project organism, click on the "Select an Organism" drop-
 
 The organism names are derived from the [NCBI taxonomy database](http://www.ncbi.nlm.nih.gov/taxonomy).
 
-To lock sample modification in the new project, check the "Lock sample modification" box. This setting will only allow the sample owner to edit any sample data.
+Clicking the "Next" buttons allows you to select samples that have been added to your cart to the project.
```

Comment:
Change `Clicking the "Next" button allows you to select samples that have been added to your cart and add them to the project.`

Reply:
Updated in 6e26a5fd570aafe0f4d87cda9e298e8069b2d680